src/Frontend/components/portal/signup/Form.js

```diff
@@ -107,12 +109,26 @@ export default class Form extends Component {
           helper={this.helper}
           value={this.helper.getValue(this.state.data.locale, 'locale')}
           options={{ de: 'Deutsch (Deutschland)', en: 'English (USA)' }} />
-        <LoadableButtonBar btnLabel={this.helper.getFormFieldAlias('button')} progress={this.state.progress} />
+          <Recaptcha
+              sitekey="6Lds4igTAAAAAJDx1KvNfznuELzNauwKu21ODAWR"
```

Comment:
to hide such keys completely before merging, I recommend to build a config file (e.g. Frontend/config/recaptcha.js) ignored by git which exports a string, nothing more. It might be generated by NPM postinstall if it doesn't exist and to test registration the key must be added there, too (this note that the key must be added also there should be added to the docs)